warehouse/query-core/src/test/java/datawave/query/jexl/visitors/QueryOptionsFromQueryVisitorTest.java

```diff
@@ -283,6 +283,36 @@ public void testNonFunctionNodesWithJunctions() throws ParseException {
         assertOption(QueryParameters.GROUP_FIELDS, "field1,field2");
     }
 
+    @Test
+    public void testLanguageExpansionCodes() throws ParseException {
+        assertResult("FOO == 'bar' && f:options('language.expansion', 'en,es')", "FOO == 'bar'");
+        assertOption(QueryParameters.LANGUAGE_EXPANSION, "en,es");
+    }
+
+    @Test
+    public void testDisableStemming() throws ParseException {
+        assertResult("FOO == 'bar' && f:options('disable.stemming', 'en,es')", "FOO == 'bar'");
+        assertOption(QueryParameters.DISABLE_STEMMING, "en,es");
+    }
+
+    @Test
+    public void testDisableLemmas() throws ParseException {
```

Comment:
Can we get some query tests that validate the Plans when disabling these pieces in addition to just making sure the parameter is set?  If they already exist somewhere I haven't found yet, disregard.